docs/usage/view-options-panel.md

```diff
@@ -0,0 +1,81 @@
+# View Options Panel
+
+![view options panel with more options expanded.](../media/view_options_panel.png)
+
+
+## Zoom to fit Trajectory
+
+This zooms to fit the trajectory of the focused path. You can also access this option through the shortcut `Control + 0` or `Command + 0`
+
+## Path Gradients
+Path Gradients are colors performed between intervals or between waypoints, depending on the specified behavior. This can be helpful for those trying to debug information quickly, or verify a constraint's effects.
```

Comment:
```suggestion
Path Gradients are colors displayed between intervals or between waypoints, depending on the specified behavior. This can be helpful for those trying to debug information quickly, or verify a constraint's effects.
```